addStudy.py

```diff
@@ -0,0 +1,101 @@
+# addStudy.py
+
+#!/usr/bin/python3
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# addStudy.py
+#
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+from include.calibrationLib import *
+
+# List all studies
+def get_studies(configuration, studName):
+    # return "List to populate"
+    sql = '''SELECT * FROM STUDY'''
+
+    return select(configuration["connection_string"], sql, {'name': None})
+
+# add studies
+def add_file(configuration, Name):
```

Comment:
Is `add_file` a good function name for something that adds a new study to a database?